### ISSUE TITLE
Default Player will not play in themoviedbhelper but in a favorite

### ISSUE THREAD

**Issue by pewkodi** [jurialmunkey/plugin.video.themoviedb.helper#221]
i have a custom plugin that is a webscraping plugin. It will not open the dialog to find links using the moviedb helper plugin but it will work if i create a favorite of the item i want to get links for. 

is there something different from creating a favorite and use the moviedb helper to open the default player?

Thanks

**Comment by pewkodi** [comment]
here is the favorite that was created:

>PlayMedia(&quot;plugin://plugin.video.themoviedb.helper/?info=play&amp;episode=2&amp;tmdb_id=19885&amp;season=4&amp;type=episode&amp;nextpage=True&quot;)

**Comment by jurialmunkey** [comment]
No there shouldn't be any difference. 
Do you mean TMDbHelper's dialog to select the player or do you mean some other dialog in your webscraping plugin?
Are you attempting to play from a widget or in the video add-ons section? 
Are you playing through the info dialog?
What skin are you using?
What are the exact steps to reproduce the issue?
What is the player file?

Also a debug log with debug logging turned on would be useful.

**Comment by pewkodi** [comment]
Steps to reproduce:
I added the json file into my userdata/addon_data folder under plugin.video.themoviedbhelper player. 
Go to the settings for mdb helper under player and select the plugin I added.
open mdb helper and search a tv show/episode. clicking on it will bring up working then stop.

If I right-click on the episode and save as favorite. It produces with the name of the episode. a thumbnail and the Playmedia tag like above.

Go to my favorites and click on the favorite I created. It will open up a dialog box and start the web scraping to produce the results it finds with the correct plugin.

The mdb helper plugin with not open the dialog window from the plugin to do the web scraping. 

The skin is confluence.

Here is the log file:

http://ix.io/2lZK

here is the json file:

http://ix.io/2lZJ
